benchmark/root_child_send_tx.go

```diff
@@ -0,0 +1,207 @@
+package benchmark
+
+import (
+	"flag"
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/e2e-polybft/framework"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/abi"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+var (
+	singleContCalcFunc   = abi.MustNewMethod("function compute(uint256 x, uint256 y) public returns (uint256)")
+	singleContGetFunc    = abi.MustNewMethod("function getValue() public returns (uint256[] memory)")
+	singleContSetFunc    = abi.MustNewMethod("function addValue(uint256 value) public")
+	multiContSetAddrFunc = abi.MustNewMethod("function setContractAddr(address _contract) public")
+	multiContFnA         = abi.MustNewMethod("function fnA() public returns (uint256)")
+	RootJSONRPC          = flag.String("rootJSONRPC", "", "JSONRPC address of the root node")
```

Comment:
Do we need these flags here? I am not a fan of setting a global variable value from a different file and package. Could we just pass some struct or arguments to the `RootChildSendTxSetUp` function.